.gitignore

```diff
@@ -141,3 +141,6 @@ h2o-docs/src/booklets/source/PythonBooklet.tui
 *.gz(busy)
 
 h2o-web/package-lock.json
+
+.tests-r.txt
```

Comment:
If we put these file in the results directory they will be automatically ignored. Storing them in the results directory would have one more benefit - they would be archived by the jenkins job and provide another info when debugging.

Reply:
good point, I will change it

Reply:
... well actually, the `results` directory is being wiped at the beginning of `run.py` which makes it unsuitable for storing configuration.
Instead, I changed the location to be `h2o-*/tests/.lookup.txt` and added that into set of artifacts archives by jenkins. So it will appear on UI, just from slightly different location.

Reply:
right!